azure-pipelines.yml

```diff
@@ -0,0 +1,66 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - '*'
+
+jobs:
+
+- job: 'Test'
+  strategy:
+    matrix:
+      Python36Linux:
+        imageName: 'ubuntu-latest'
+        python.version: '3.6'
+      Python36Windows:
+        imageName: 'windows-2019'
+        python.version: '3.6'
+      Python37Windows:
+        imageName: 'windows-latest'
+        python.version: '3.7'
+      Python38Linux:
+        imageName: 'ubuntu-latest'
+        python.version: '3.8'
+      Python39Mac:
+        imageName: 'macos-latest'
+        python.version: '3.9'
+      Python310Linux:
+        imageName: 'ubuntu-latest'
+        python.version: '3.10'
+      Python310Windows:
+        imageName: 'windows-latest'
+        python.version: '3.10'
+      Python310Mac:
+        imageName: 'macos-latest'
+        python.version: '3.10'
+      Python311Linux:
+        imageName: 'ubuntu-latest'
+        python.version: '3.11'
+      Python311Windows:
+        imageName: 'windows-latest'
+        python.version: '3.11'
+      Python311Mac:
+        imageName: 'macos-latest'
+        python.version: '3.11'
+    maxParallel: 4
+  pool:
+    vmImage: $(imageName)
+
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(python.version)'
+      architecture: 'x64'
+
+  - script: |
+      python -m pip install -U build pip setuptools
+      pip install -U -r requirements.txt
+    displayName: 'Install dependencies'
+
+  - task: DeleteFiles@1
+    inputs:
+      contents: 'openai-prodigy-recipes'
+    displayName: 'Delete source directory'
```

Comment:
What's with this part?

Reply:
Oh we don't need this bit if we're not building from sdist, I'll remove it.